site/content/en/faq.md

```diff
@@ -378,6 +210,39 @@ Exceptions are treated almost identically to licenses. In order to combine a
 license with an exception, you mark a file with the following tag:
 `SPDX-License-Identifier: GPL-3.0-or-later WITH GCC-exception-3.1`.
 
+## How does license compatibility work? {#license-compatibility}
+
+Free Software licenses are all different. Many licenses are compatible, meaning
```

Comment:
```suggestion
Free Software licenses often differ in detail. Many licenses are compatible, meaning
```

Many are more or less the same (e.g. think of all MIT/ISC/BSD/X11/…-likes).

And even if you compare e.g. AGPL and 0BSD, in the grand scheme of things they both offer the four freedoms, which is quite a huge and important part. The details can be quite important, but they remain details.